dali/pipeline/operator/error_reporting.cc

```diff
@@ -96,7 +96,7 @@ void PropagateError(ErrorInfo error) {
   }
 }
 
-std::string GetErrorContextMessage(const OpSpec &spec) {
+std::string GetErrorContextMessage(const OpSpec &spec, const std::string &message_name) {
```

Comment:
How about using a `string_view` in such functions?
```suggestion
std::string GetErrorContextMessage(const OpSpec &spec, std::string_view message_name) {
```

Reply:
done